godot-macros/src/virtual_method_callback.rs

```diff
@@ -0,0 +1,41 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+use proc_macro2::TokenStream as TokenStream2;
+use quote::quote;
+
+use crate::method_common::*;
+
+pub fn transform(method_def: MethodDef) -> TokenStream2 {
+    let method_signature = method_def.sig;
+
+    let instance_ptr = quote! {instance_ptr};
+    let args = quote! {args};
+    let ret = quote! {ret};
```

Comment:
Please make sure to use existing `quote!` syntax with spacing:
```suggestion
    let instance_ptr = quote! { instance_ptr };
    let args = quote! { args };
    let ret = quote! { ret };
```

But I don't really see the point of defining those variables at all -- why not just inline them in the other `quote!` that includes them?